common/src/main/java/com/wynntils/screens/base/WynntilsContainerScreen.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.base;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.utils.mc.McUtils;
+import net.minecraft.ChatFormatting;
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+
+public abstract class WynntilsContainerScreen<T extends AbstractContainerMenu> extends AbstractContainerScreen<T> {
```

Comment:
This duplication of functionality is a bit annoying. :( But I guess there is not much to do about it, as long as we are stuck with the class hierarchy of Minecraft, and a Java language that do no do multiple inheritance... :-/

Reply:
If anything, maybe you can make the failure handler in WynntilsScreen public static, and call that one, instead of duplicating it.

Reply:
Nah, on second thought, it's not worth it either. Just let this be.